hieradata/common.yaml

```diff
@@ -1 +1,19 @@
-classes: []
+classes:
+    - ocf::apt
+    - ocf::auth
+    - ocf::autologout
+    - ocf::groups
+    - ocf::kerberos
+    - ocf::ldap
+    - ocf::locale
+    - ocf::motd
+    - ocf::munin::node
+    - ocf::networking
+    - ocf::packages
+    - ocf::puppet
+    - ocf::rootpw
+    - ocf::staff_users
+    - ocf::systemd
+    - ocf::utils
+
+packages: {}
```

Comment:
Meh, seems like this shouldn't be duplicated both here and in the `ocf::packages` class. I can just see people adding packages here instead of in the correct class, when it would be much better to have that all in the manifests if possible.